tests/test_live_events.py

```diff
@@ -38,7 +38,7 @@ def setUp(self):
         self.strategy_pv = PVStrategy(
             panel_count=3, initial_selling_rate=34, final_selling_rate=12,
             fit_to_limit=False, update_interval=6, energy_rate_decrease_per_update=4,
-            max_panel_power_W=432)
+            capacity_kW=432)
```

Comment:
Nit: probably unnecessary, but should it be converted to kW?

Reply:
The same applies to the rest of the file. It's probably not needed since all computations will be scaled up and remain consistent, I assume.

Reply:
https://github.com/gridsingularity/d3a/pull/1192#discussion_r684089286